index.html

```diff
@@ -0,0 +1,487 @@
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+<meta charset="UTF-8">

```

Comment:
Hello @nived-dev , I am very impressed that you have submitted the code within 6 days but the favicon image cannot be seen **try checking the code again.** 
Anyway considering your work, I am considering this merge request